### PR TITLE
Add add_raw_disk_to_vm sample

### PR DESCRIPTION
**Pull request by jkufro**
New sample derived from add_disk_to_vm example that allows adding a raw disk to an already existing VM. I have tested on vmWare ESXi 6.7.

### CODE REVIEW

File: samples/add_raw_disk_to_vm.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+import atexit
+import argparse
+import getpass
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description='Arguments for talking to vCenter')
+
+    parser.add_argument('-s', '--host',
```
Comment (pgbidkar) [inline samples/add_raw_disk_to_vm.py]: Please use library argument builder https://github.com/vmware/pyvmomi-community-samples/blob/83c8bc362d3c3eaec665228618b62a958d0752a7/samples/tools/cli.py#L25

File: samples/add_raw_disk_to_vm.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+                        required=False,
+                        default='virtualMode',
+                        choices=['physicalMode', 'virtualMode'],
+                        action='store',
+                        help=('See vim.vm.device.VirtualDiskOption.'
+                              'CompatibilityMode documentation.'))
+
+    args = parser.parse_args()
+
+    if not args.password:
```
Comment (pgbidkar) [inline samples/add_raw_disk_to_vm.py]: Please use Library method for this as well

File: samples/add_raw_disk_to_vm.py [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+        disk_spec.device.controllerKey = controller.key
+        spec.deviceChange = [disk_spec]
+        WaitForTasks([vm.ReconfigVM_Task(spec=spec)], si=si)
+        print "Raw disk added to %s" % (vm.config.name)
+
+
+def main():
+    args = get_args()
+
+    # connect this thing
```
Comment (pgbidkar) [inline samples/add_raw_disk_to_vm.py]: Please make comment more clear.

**Comment by pgbidkar** [comment]
Hi @jkufro , Thanks for contributing

**Comment by jkufro** [comment]
@pgbidkar thanks for your feedback. Please review changes.

**Comment by kumahesh** [comment]
I have reviewed the code. It looks good. Could you run the sample and post the observation or results or screenshots of added RDM to vm (vcenter screenshot)here? then we are good to merge it.

**Comment by jkufro** [comment]
@kumahesh see below.

Screen Shot 2019-07-23 at 8 01 15 AM
Screen Shot 2019-07-23 at 8 04 02 AM

**Comment by pgbidkar** [comment]
Hi @jkufro , Thanks for contributing

**Pull request merged**